beta/src/content/learn/passing-props-to-a-component.md

```diff
@@ -11,16 +11,16 @@ React components use *props* to communicate with each other. Every parent compon
 <YouWillLearn>
 
 * How to pass props to a component
-* How to read props from a component
+* How to read props in a component
 * How to specify default values for props
 * How to pass some JSX to a component
 * How props change over time
 
 </YouWillLearn>
 
-## Familiar props {/*familiar-props*/}
+## Familiar-looking props {/*familiar-props*/}
 
-Props are the information that you pass to a JSX tag. For example, `className`, `src`, `alt`, `width`, and `height` are some of the props you can pass to an `<img>`:
+Props are the information that you pass to a React element. For example, `className`, `src`, `alt`, `width`, and `height` are some of the props you can pass to an `img` element:
```

Comment:
We're deemphasizing the concept of "elements" in the new docs. They're kind of fading into background a bit as an implementation detail. E.g. JSX transform doesn't even call `createElement`. We want to discourage their introspection too. So I'm not sure we'd like to rely on this terminology.

Reply:
Can I see a draft of the new terminology somewhere? As element, tag, and attribute are HTML/XML terminology, I'd expect them to be used with JSX and React as well, regardless of the implementation choices. I currently think of tags and attributes as markup syntax and of elements as component instances. Then, props (encompassing `children` and lacking `key`) are what the component instance receives. Am I far off?